src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -544,6 +573,19 @@ def _init_weights(self, module):
         if isinstance(module, (nn.Linear, nn.Conv1d)) and module.bias is not None:
             module.bias.data.zero_()
 
+    def _get_feat_extract_output_lengths(self, input_lengths: torch.LongTensor):
```

Comment:
This function is quite important for speech since there is often a mismatch in sequence input length and model output length due to the use of `stride` in the convolutional layers. However, the model needs to know the "correct" feature extraction output length when treating padded input.